components/list/list.js

```diff
@@ -64,13 +71,65 @@ class List extends LitElement {
 		return [layout, specialCases];
 	}
 
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+		this._checked = {};
+		this.addEventListener('d2l-list-item-checked', this._onlistItemChecked.bind(this));
+	}
+
+	itemsChecked() {
+		return Object.keys(this._checked).filter(ref => this._checked[ref]);
+	}
+
 	render() {
 		return html`
 			<div role="list" class="d2l-list-container">
 				<slot></slot>
 			</div>
 		`;
 	}
+
+	selectionState() {
```

Comment:
Similarly, I'd call this `getSelectionState()`.

Reply:
- [ ] Update name to `getSelectionState()`